less/about/preferences.less

```diff
@@ -862,17 +946,18 @@ div.nextPaymentSubmission {
       color: @darkGray;
       font-weight: bold;
       font-size: 15px;
-      margin-bottom: 15px;
-      padding-right: 25px;
-    }
-    .settingsListSubTitle {
-      color: @darkGray;
-      clear: both;
-      font-weight: normal;
-      font-size: 14px;
-      font-style: italic;
-      line-height: 1.3em;
+      margin: 0;
+
+      &.subTitle {
```

Comment:
`.settingsListSubTitle` was replaced with `.settingsListTitle .subTitle`, which can be more easily maintainned.